documentation/modules/rn-2.5.adoc

```diff
@@ -124,40 +130,113 @@ When migrating VMs that are installed with RHEL 9 as guest operating system from
 
 When adding an OVA provider, the error message `ConnectionTestFailed` may instantly appear, although the provider is created successfully. If the message does not disappear after a few minutes and the provider status does not move to `Ready`, this means that the `ova server pod creation` has failed. link:https://issues.redhat.com/browse/MTV-671[(MTV-671)]
 
-For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/browse/MTV-562?filter=12419159[Known Issues] in Jira.
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/browse/MTV-740?filter=12424645[Known Issues] in Jira.
```

Comment:
I didn't notice this before but there's an issue with this view, take for instance https://issues.redhat.com/browse/MTV-791 that appears there - it has affected-version=2.5.1 so it is supposed to come up by this search but its clone (https://issues.redhat.com/browse/MTV-812) is fixed in this release

we can add `and issueLinkType != "is cloned by"` to the query, but this will filter out results whose clone is not resolved yet, find the status of the clone becomes complicated..

Reply:
changed to 

```
project = MTV AND issuetype = Bug AND status not in (Verified, "Release Pending", Closed) AND component != Documentation AND (affectedVersion  <= 2.5.3 and affectedVersion >= 2.5.0)  AND issueLinkType != "is cloned by" ORDER BY priority DESC
```